cmake/Platform/Mac/runtime_dependencies_mac.cmake.in

```diff
@@ -187,7 +187,7 @@ if(@target_file_dir@ MATCHES ".app/Contents/MacOS")
             )
             file(GLOB_RECURSE exe_file_list "${bundle_path}/Contents/Frameworks/Python.framework/**/*.exe")
             if(exe_file_list)
-                file(REMOVE_RECURSE "${exe_file_list}")
+                file(REMOVE_RECURSE ${exe_file_list})
```

Comment:
make sense, 'exe_file_list' is a list, so this was quoting the list entirely, turning it from a list to a single item. Would we want to do a list transform instead to quote individual items in exe_file_list?

Reply:
I don't think so.  As long as cmake itself interprets each element in the list as a separate file, it usually handles spaces without issue since its separator is a semicolon, not a space.  I guess it kinda depends on what the internal implementation of that function is in cmake C++.  But usually they're fine.

Reply:
ah, this was my bad, yes, it shouldnt have the quotes
I added quotes to certain places to fix space issues, but this is a list.
Here it cannot be with quotes because the list needs to be passed as multiple parameters. With quotes, is passed as one parameter that contains a list. `file(REMOVE_RECURSE` expects each parameter to be a path (and not a list)